models/threshold_test.go

```diff
@@ -393,6 +393,158 @@ func Test_Threshold_ToInternal(t *testing.T) {
 				},
 			},
 		},
+		{
```

Comment:
Perhaps just make sure that there's a similar test for HAL.

Reply:
There is if you scroll down ;)

Reply:
Do you expect me to expand the full diff of the file? ;)